portal-api/src/v1/controllers/deal-status/send-status-update-emails.js

```diff
@@ -1,9 +1,11 @@
 const sendEmail = require('../../email');
 
 const sendEmailsToOwningBanks = async (templateId, emailVariables, owningBankEmails) => {
-  owningBankEmails.forEach(async (email) => {
-    await sendEmail(templateId, email, emailVariables);
-  });
+  await Promise.all(
+    owningBankEmails.map(async (email) => {
+      await sendEmail(templateId, email, emailVariables);
+    }),
```

Comment:
Maybe `owingBankEmails.map((email) => sendEmail(templateId, email, emailVariables))` for conciseness?